### DISCUSSION TITLE
Is there a way to pack extracted file content

### DISCUSSION THREAD

**Discussion by Terixxx** [sn4k3/UVtools#821] [Q&A] [answered]
Hello,
I have question about extracted file content. I need to try to manually edit configuration file and after that to pack it in the requred file format. Is there option like this?
Thanks

**Comment by sn4k3** [comment, marked as answer]
There's no option. Unpack contents are sorted and managed by UVtools. It does not represent a dump of the real file.
But which settings are you trying to edit that you can't via the UI?

**Reply by Terixxx** [reply]
I need to change IntelligentMode or AdvancedMode.

**Reply by sn4k3** [reply]
~~AdvancedMode you can't change because UVtools auto set that parameter based on current settings (Off if not required or On if required)~~
IntelligentMode however you can set on UI using the Terminal to do via UI, go to File - Terminal. Send the command: `((PhotonWorkshopFile)SlicerFile).HeaderSettings.IntelligentMode = true`. Then Save the file.
Note: IntelligentMode will set to false if you change exposure time via UVtools.

Edit: AdvancedMode can also be edited, as same way as IntelligentMode as long you don't edit Lift Height for TSMC or it will set to true/false depending on settings.
`((PhotonWorkshopFile)SlicerFile).HeaderSettings.AdvancedMode = 1`

**Reply by Terixxx** [reply]
Am I need to save file after that?

**Reply by sn4k3** [reply]
Yes

**Reply by Terixxx** [reply]
Thank you very much! This solved all my issues
